dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyTreeFormatter.java

```diff
@@ -105,7 +105,7 @@ private static void formatDependencyPathTree(
    * @param dependencyPaths dependency path instances without assuming any order
    * @return ListMultiMap representing a Maven dependency tree
```

Comment:
If we stick with this representation for now, we should document how the tree is represented as a list multimap.

Reply:
Added description in Javadoc.